content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -722,12 +722,7 @@ and need persistent storage, it is recommended that you use the following patter
     `persistentVolumeClaim.storageClassName` field.
     This will cause the PVC to match the right storage
     class if the cluster has StorageClasses enabled by the admin.
-  - If the user does not provide a storage class name, leave the
-    `persistentVolumeClaim.storageClassName` field as nil.
-    - This will cause a PV to be automatically provisioned for the user with
-      the default StorageClass in the cluster. Many cluster environments have
-      a default StorageClass installed, or administrators can create their own
-      default StorageClass.
+  - If the user does not provide a storage class name, leave the `persistentVolumeClaim.storageClassName` field as nil.This will cause a     PV to be automatically provisioned for the user with the default StorageClass in the cluster. Many cluster environments have a           default StorageClass installed, or administrators can create their own default StorageClass.
```

Comment:
Thanks for doing this! I have just one minor suggestion. 

It looks like when you merged the bullets, some spacing issues were introduced and the line breaks were removed. A space should be added between "nil." and "This" and it's probably best to break this into multiple lines for readability, similar to the original formatting.

Reply:
thanks for the suggestion.I will change the formatting .

Reply:
@geekygirldawn  i have made changes as per your suggestion. Please review it.